tests/e2e/test_operator.py

```diff
@@ -35,6 +35,12 @@ def test_001(self):
             "pdb": ["single"],
         },
     )
+    with Then("I delete namespace"):
```

Comment:
if we have create_test_namespace()

why we make lot of copy\paste code instead of create
delete_test_namespace() ?